### DISCUSSION TITLE
Bug: Scroll bar with long expandable code

### DISCUSSION THREAD

**Discussion by bkbilly** [GitbookIO/community#243] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

I have created an expandable with a huge yaml code that I want it hidden so that people don't have to scroll all the way to the bottom. The problem is that when it is collapsed, the scroll bar stays at the size of the expanded code.

You can check the problem here:
https://bkbilly.gitbook.io/lnxlink/examples#keys-send

**Comment by addisonschultz** [comment]
Hey there! Thanks for reporting - I've filed a bug for us to hop on right away. 

Just to check something - if you remove the expandable, it works as expected? Also - if you have long content that isn't a code block, does it work? 

Having trouble reproducing this one, but we'll investigate to see what's happening

**Comment by addisonschultz** [comment]
Thank you for sending us an example of this in production by the way! It's very helpful 🙏
